### PR TITLE
Add possibility to add nrpe check from other repos

### PR DESCRIPTION
**Pull request by oscarkraemer**
It is now possible to add nrpe check from other git repos and it should be
possible to easily extend this with other src types than git.

CSCWOOD-16

### CODE REVIEW

**Comment by oscarkraemer** [comment]
#14 Should be merged before this. Just to make clear difference between the changes.

**Comment by martbhell** [comment]
One more to unsubscribe from!
Nice to see it's still being used :)

See https://github.com/CSCfi/ansible-role-nrpe-plugins for some examples of other ways like get_url and pip.

File: tasks/add_additional_nrpe_checks.yml
Comment (junousi) [inline tasks/add_additional_nrpe_checks.yml:14]: Here I had a moments hesitation, should it be single 'quoted' or are there some corner cases where some weird variable expansion can happen. OTOH the way it's currently used looks fine.

File: templates/check_nagios.cfg.j2
Comment (junousi) [inline templates/check_nagios.cfg.j2]: Would have been nicer to retain uniform structure in the template, i.e. still hardcode `command` here and get the dynamic contents from the hash, since now it's decoupling the template structure into two places instead of one neat place. I'm going to approve this and leave it for the author to decide if this should be made more beautiful : )

Comment (junousi) [reply templates/check_nagios.cfg.j2]: Ah! We don't have the approval step in this repo. I'll merge this after comment from @oscarkraemer .

File: tasks/add_additional_nrpe_checks.yml
Comment (oscarkraemer) [reply tasks/add_additional_nrpe_checks.yml:14]: I think that it is in bash and puppet where we care about single vs. double quotation.

**Review by junousi — APPROVED** [review]
Approving with comments.

File: templates/check_nagios.cfg.j2
Comment (oscarkraemer) [reply templates/check_nagios.cfg.j2]: I will make improvement to this.

Comment (oscarkraemer) [reply templates/check_nagios.cfg.j2]: Okay, I changed this. I think the change is an improvement overall. My master plan is to someday make a breaking change where I remove `nagios_plugins` and `opsview_plugins` variables.

**Pull request merged**
